src/modules/cluster/cluster-detail/cluster-detail.js

```diff
@@ -6,34 +6,26 @@
     app.controller("clusterDetailController", clusterDetailController);
 
     /*@ngInject*/
-    function clusterDetailController($state, $stateParams, utils) {
+    function clusterDetailController($state, $stateParams, utils, $scope) {
 
         var vm = this;
-        vm.tabList = {};
+        vm.clusterName;
+        vm.tabList = {"Host": 1};
         vm.tabName = "";
         vm.setTab = setTab;
         vm.isTabSet = isTabSet;
 
-        vm.clusterId = $stateParams.clusterId;
-
-        utils.getObjectList("Cluster").then(function(list) {
-            vm.cluster = list.filter(function(cluster) {
-                return cluster.cluster_id === vm.clusterId;
-            })[0];
-
-            /* TODO:- Cluster specific tab info should come from API itself 
-            Setting up the tab based on ceph/gluster */
-            var clusterType = vm.cluster.sds_name.toLowerCase();
-            if( clusterType.indexOf("gluster") !== -1 ) {
-                vm.tabName = "Volume";
-            } else if( clusterType.indexOf("ceph") !== -1 ) {
-                vm.tabName = "Pool";
-            }
-
-            vm.tabList[vm.tabName] = 1;
-            vm.activeTab = vm.tabList[vm.tabName];
-            
-        });
+        $scope.clusterId = $stateParams.clusterId;
```

Comment:
@kamleshverma1 You can add the clusterId on vm instance as rest of the properties are on vm instance.